nodejs/eks/cluster.ts

```diff
@@ -665,6 +684,41 @@ function createInstanceRoleMapping(arn: pulumi.Input<string>): RoleMapping {
     };
 }
 
+/**
+ * Create an HTTP Agent for use with HTTP(S) requests.
+ * Using a proxy is supported.
+ */
+function createHttpAgent(proxy?: string): http.Agent {
```

Comment:
Such a shame we need this - but looks like Node.js (and even libraries like `node-fetch`) don't build this in :-(.